Cargo.toml

```diff
@@ -56,6 +54,8 @@ sp-runtime-interface = { version = "28.0.0", features = ["std"] }
 # Local
 drink = { path = "crates/drink/drink" }
 ink_sandbox = { path = "crates/ink-sandbox" }
+pallet-api = { path = "../pop-node/pallets/api" }
```

Comment:
A reminder that these will need to be updated to the actual repos before this is merged.

Reply:
https://github.com/r0gue-io/pop-drink/issues/20